Bottom: 6d29d42a529305eee9ce083d3f7d91f2f774e33a
Top:    73d4099ac61bc76a7511043f72d9eec90a48fc0b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 16:23:43 -0500

Refresh of gitlab-10.8

---
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index f09245f0..61f19ea8 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -118,22 +118,12 @@
     state: started
 
 - name: Wait for gitlab to be fully running
-  uri:
-    url: 'https://{{ gitlab_hostname }}/users/sign_in'
-  register: health
-  retries: 10
-  delay: 30
-  until: health is success
-  when: gitlab_version is version('8.17', '<')
-
-- name: Wait for a container to become healthy
   docker_health:
-    name: "{{gitlab_container_name}}"
+    name: '{{ gitlab_container_name }}'
   register: health
-  retries: 10
-  delay: 30
+  retries: 60
+  delay: 5
   until: health.Health.Status == "healthy"
-  when: gitlab_version is version('8.17', '>=')
 
 # *****************************************************************************
 # restore?
diff --git a/roles/docker-gitlab/tasks/restore.yml b/roles/docker-gitlab/tasks/restore.yml
index fc564206..e01594c8 100644
--- a/roles/docker-gitlab/tasks/restore.yml
+++ b/roles/docker-gitlab/tasks/restore.yml
@@ -114,6 +114,14 @@
        user.password_confirmation="{{ gitlab_root_password }}";
        user.save!'
 
+- name: Wait for gitlab to be fully running
+  docker_health:
+    name: '{{ gitlab_container_name }}'
+  register: health
+  retries: 60
+  delay: 5
+  until: health.Health.Status == "healthy"
+
 # *****************************************************************************
 # cleanup
